spec/lib/feature_management_spec.rb

```diff
@@ -181,49 +181,26 @@
     end
   end
 
-  describe '.no_pii_mode?' do
-    let(:proofing_vendor) { :mock }
-    let(:enable_identity_verification) { false }
-
-    before do
-      allow_any_instance_of(Figaro.env).to receive(:profile_proofing_vendor).
-        and_return(proofing_vendor)
-      allow(Figaro.env).to receive(:enable_identity_verification).
-        and_return(enable_identity_verification.to_json)
-    end
-
-    subject(:no_pii_mode?) { FeatureManagement.no_pii_mode? }
-
-    context 'with mock ID-proofing vendors' do
-      let(:proofing_vendor) { :mock }
-
-      context 'with identity verification enabled' do
-        let(:enable_identity_verification) { true }
-
-        it { expect(no_pii_mode?).to eq(true) }
-      end
-
-      context 'with identity verification disabled' do
-        let(:enable_identity_verification) { false }
+  describe '.fake_banner_mode?' do
+    subject(:fake_banner_mode?) { FeatureManagement.fake_banner_mode? }
```

Comment:
This subject doesn't seem to be used. Can we remove it?

Reply:
DONE